src/web/mjs/connectors/Manatoki.mjs

```diff
@@ -0,0 +1,54 @@
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
+
+export default class Manatoki extends GnuBoard5BootstrapBasic2 {
+
+    constructor() {
+        super();
+        super.id = 'manatoki82';
+        super.label = 'Manatoki';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://manatoki82.net';
```

Comment:
This URL is probably going to be changed every 2 weeks.
See `NewToki` or `ToonKor` how this can be handled with `canHandleURI` and `_initializeConnector` methods.

Reply:
that's nice to know, but i don't know how it will change.